bc_obps/service/transfer_event_service.py

```diff
@@ -257,11 +257,14 @@ def _process_facilities_transfer(cls, event: TransferEvent, user_guid: UUID) ->
             # update the facility's operation
             FacilityService.update_operation_for_facility(user_guid=user_guid, facility=facility, operation_id=event.to_operation.id)  # type: ignore # we are sure that operation is not None
 
+    # @transaction.atomic()
     @classmethod
+    @transaction.atomic()
     def _process_operation_transfer(cls, event: TransferEvent, user_guid: UUID) -> None:
         """
-        Process an operation transfer event. Updates the timelines for the associated operation.
+        Process an operation transfer event. Updates the timelines for the associated operation. Deletes operation_contacts that belong to the original operator.
```

Comment:
I know I'm being pedantic, but I'm concerned that this last sentence could introduce developer confusion. We're only removing the relationship between an operation and its contacts, not deleting contacts, which I'm concerned this sentence might be interpreted as. Could you please reword this to make it clearer?